test/find.cpp

```diff
@@ -156,19 +156,12 @@ TEST(FindTests, ByPath)
     ASSERT_EQ(count, 34);
 
     ASSERT_EQ(range5.begin()->getIndex(), 0);
-    count = 0;
-    for(auto& entry: range5) {
-      count++;
-    }
+    count = (--range5.end())->getIndex() - range5.begin()->getIndex() + 1;
```

Comment:
You are not testing that the range contains 118 entries.
You are testing that the index of the last entry is 118 more than the index of the first entry.
But somehow (a bug), the range could contains only the first and the last entry.

Reply:
Understood. I guess it's better to follow the same format as the rest of the tests. Will switch back to a for loop printing the paths of the entries.